src/Sylius/Component/Core/Distributor/ProportionalIntegerDistributor.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sylius\Component\Core\Distributor;
```

Comment:
Agreed 👍 However, it's placed here because previously implemented ``IntegerDistributor`` has been placed here too. Maybe we can move them together in separate PR to avoid making them sad and alone?